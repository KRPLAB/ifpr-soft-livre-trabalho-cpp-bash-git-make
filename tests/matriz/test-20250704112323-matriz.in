4
58 15 83 28
81 71 70 36
23 59 68 44
44 27 60 48
38 60 99 91
14 53 75 49
14 37 65 92
63 61 6 16
a
