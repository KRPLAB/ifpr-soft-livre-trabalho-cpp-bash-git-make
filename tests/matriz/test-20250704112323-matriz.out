Dimensoes da matriz:
Matriz
96	75	182	119	
95	124	145	85	
37	96	133	136	
107	88	66	64	
